pkg/api/validate/plugin.go

```diff
@@ -24,6 +24,10 @@ func (v *PluginAPIValidator) Validate(t *pluginapi.Config) (errs []error) {
 	if !clusterVersion.MatchString(t.ClusterVersion) {
 		errs = append(errs, fmt.Errorf("invalid ClusterVersion %q", t.ClusterVersion))
 	}
+	// validate ClusterLogLevel
+	if t.ClusterLogLevel%2 != 0 || (t.ClusterLogLevel < 2 || t.ClusterLogLevel > 8) {
```

Comment:
0 is a valid clusterloglevel.  Any reason we are excluding it?

Reply:
rationale was that setting 0 wouldn't be helpful to a cluster admin. I'll revert this